CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Changed
 
 ### Fixed
+- Issue where users were unable to exit YouTube with remote back button (#1542)
```

Comment:
Not your fault: the release date for 3.1.3 is inaccurate and the Changed header should only be present if there are items under it